client/src/utils/filePermissions.js

```diff
@@ -34,6 +34,12 @@ export default function getFilePermissions(name, { files, scheme, api }) {
         files[0].format !== 'folder' &&
         api !== 'googledrive'
       );
+    case 'isFolderSelected':
```

Comment:
> dialog message is intended for folders only.

The dialog message is still intended for folders only but this check would be true if users selected two text files and a single folder.  In that case the message would be off.

Reply:
Please see this message and @rstijerina reply:
https://github.com/TACC/Core-Portal/pull/425#discussion_r652851009

Reply:
_I tested two files and one folder; it gives the message._

_I think our users will understand this message appears because they selected a folder._

_Clarity may be nice. Feel free to request new UX from or propose new UX to Design in #cep-design._